hadoop-ozone/interface-storage/src/main/java/org/apache/hadoop/ozone/om/ratis/TransactionInfo.java

```diff
@@ -52,11 +52,23 @@ private OMTransactionInfo(String transactionInfo) {
     transactionIndex = Long.parseLong(tInfo[1]);
   }
 
-  private OMTransactionInfo(long currentTerm, long transactionIndex) {
+  private TransactionInfo(long currentTerm, long transactionIndex) {
     this.term = currentTerm;
     this.transactionIndex = transactionIndex;
   }
 
+  public boolean isInitialized() {
+    return transactionIndex == -1 && term == 0;
```

Comment:
This function is not being called anywhere. 
Also, if initialized, then txIndex should not be equal to -1, right?